_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,465 @@
+---
+layout:       element
+title:        Forms
+date:         2015-05-12 16:02:35
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- forms
+
+slug:         forms
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/_src/edx-pl/sass/components/_forms.scss"
+
+description:  These examples demonstrate how to use form controls within your application.  We've grouped the form controls by type and for each type, provided proper CSS classes to achieve the desired visual presentation. In some cases, we've created optional classes which enhance the presentation. All of the examples have been tested to be accessible. Theming is available as well so form controls can match your institutions look and feel.
+
+info: Forms allow users to interact with the interface, set preferences, and enter any necessary personal information. They also form the base of individual exercises within each class. The examples below demonstrate how to use each type of form control and provides any applicable display options. Note the markup in each case to get a sense of semantics and structure.
+---
+
+<h4 class="hd-6 example-set-hd">Text inputs</h4>
+<div class="example-set">
+    <fieldset class="form-group">
```

Comment:
NIT: should we have a ``<form class="form">`` element in these examples for better context use?